public/constants/pathNames.js

```diff
@@ -0,0 +1,7 @@
+const LANDING = '/';
+const NOW_PLAYING = '/now-playing';
+const SONGS = '/songs';
+const PEOPLE = '/people';
+const MESSAGES = '/messages';
+
+export { LANDING, NOW_PLAYING, SONGS, PEOPLE, MESSAGES };
```

Comment:
Following up with above suggestion^
```suggestion
const LANDING = '/';
const LOGIN = '/api/login';
const NOW_PLAYING = '/now-playing';
const SONGS = '/songs';
const PEOPLE = '/people';
const MESSAGES = '/messages';

export { LANDING, LOGIN, NOW_PLAYING, SONGS, PEOPLE, MESSAGES };
```